frontend/src/container/AlertHistory/AlertPopover/AlertPopover.tsx

```diff
@@ -1,41 +1,113 @@
+import './AlertPopover.styles.scss';
+
 import { Popover } from 'antd';
 import LogsIcon from 'assets/AlertHistory/LogsIcon';
+import ROUTES from 'constants/routes';
+import { useIsDarkMode } from 'hooks/useDarkMode';
 import { DraftingCompass } from 'lucide-react';
 import React from 'react';
+import { Link } from 'react-router-dom';
 
-type Props = { children: React.ReactNode };
+type Props = {
+	children: React.ReactNode;
+	relatedTracesLink?: string;
+	relatedLogsLink?: string;
+};
 
-function PopoverContent(): JSX.Element {
+function PopoverContent({
+	relatedTracesLink,
+	relatedLogsLink,
+}: {
+	relatedTracesLink?: Props['relatedTracesLink'];
+	relatedLogsLink?: Props['relatedLogsLink'];
+}): JSX.Element {
+	const isDarkMode = useIsDarkMode();
 	return (
 		<div className="contributor-row-popover-buttons">
-			<div className="contributor-row-popover-buttons__button">
-				<div className="icon">
-					<LogsIcon />
-				</div>
-				<div className="text">View Logs</div>
-			</div>
-			<div className="contributor-row-popover-buttons__button">
-				<div className="icon">
-					<DraftingCompass size={14} color="var(--text-vanilla-400)" />
-				</div>
-				<div className="text">View Traces</div>
-			</div>
+			{!!relatedTracesLink && (
+				<Link
+					to={`${ROUTES.LOGS_EXPLORER}?${relatedTracesLink}`}
+					className="contributor-row-popover-buttons__button"
+				>
+					<div className="icon">
+						<LogsIcon />
+					</div>
+					<div className="text">View Logs</div>
+				</Link>
+			)}
+			{!!relatedLogsLink && (
+				<Link
+					to={`${ROUTES.TRACES_EXPLORER}?${relatedLogsLink}`}
+					className="contributor-row-popover-buttons__button"
+				>
+					<div className="icon">
+						<DraftingCompass
+							size={14}
+							color={isDarkMode ? 'var(--bg-vanilla-400)' : 'var(--text-ink-400'}
+						/>
+					</div>
+					<div className="text">View Traces</div>
+				</Link>
+			)}
 		</div>
 	);
 }
-function AlertPopover({ children }: Props): JSX.Element {
+PopoverContent.defaultProps = {
+	relatedTracesLink: '',
+	relatedLogsLink: '',
+};
+
+function AlertPopover({
+	children,
+	relatedTracesLink,
+	relatedLogsLink,
+}: Props): JSX.Element {
 	return (
-		<Popover
-			showArrow={false}
-			placement="bottom"
-			color="linear-gradient(139deg, rgba(18, 19, 23, 1) 0%, rgba(18, 19, 23, 1) 98.68%)"
-			destroyTooltipOnHide
-			content={<PopoverContent />}
-			trigger="click"
-		>
-			{children}
-		</Popover>
+		<div className="alert-popover">
+			<Popover
+				showArrow={false}
+				placement="bottom"
+				color="linear-gradient(139deg, rgba(18, 19, 23, 1) 0%, rgba(18, 19, 23, 1) 98.68%)"
```

Comment:
we can pass a class and set styles in the class

Reply:
I tried that, but it wasn't changing the background of popover.